src/production/models.py

```diff
@@ -66,3 +66,13 @@ def status(self):
                 self.completed.strftime("%Y-%m-%d %H:%M"))}
         elif self.completed and self.editor_reviewed:
             return {'slug': 'closed', 'friendly': 'Task closed'}
+
+    @property
+    def get_status_code(self):
```

Comment:
If it is a property it should probably be named `status_code` instead

Reply:
Fixed.